content/blog/2023/09/kubewarden-1.7-release.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Kubewarden 1.7.0 release
+authors:
+  - Flavio Castelli
+date: 2023-09-12
+---
+
+Today we are pleased to announce the release of Kubewarden 1.7.0! 🎊 🥳
+Aside from the bug and stability fixes, this release is packed with new features.
+This post highlights the main changes, detailed blog entries will come in the next weeks covering each feature in depth.
+
+## Audit Scanner
+
+A new component has been added to the Kubewarden stack. Its name is `audit-scanner` and
+it allows administrators to assess the compliance level of the clusters secured by Kubewarden.
+
+At regular intervals, or on demand, the audit scanner inspects all the resources defined
+inside of your Kubernetes cluster and identifies the ones that are not compliant with the
+policies that are currently enforced.
+
+Why is this useful? The number, types and configurations of the policies enforced on a cluster
+vary over its lifetime. A resource that was considered compliant some weeks ago, today
+might not be valid anymore since the introduction of a new policy.
```

Comment:
```suggestion
Why is this useful? The number, types and configurations of the policies enforced on a cluster
vary over its lifetime. 
A resource that was considered compliant recently may no longer be, since the introduction of a new policy.
```